test/integration/fixture_image_simple_test.go

```diff
@@ -119,6 +128,15 @@ func TestSimpleImage(t *testing.T) {
 
 	for _, c := range simpleImageTestCases {
 		t.Run(c.source, func(t *testing.T) {
+			if runtime.GOOS != "linux" {
```

Comment:
I ran into issues testing linux-only sources so made adjustments to skip tests when it's not a valid environment